### PR TITLE
feat: Add autologin option to Dummy provider

### PR DESCRIPTION
**Pull request by froboy**
**Related Issue/Ticket:**

Add option for dummy provider to "auto-login", providing a kind of "demo mode" for VY

## Steps to test:

- [ ] Observe new option on Dummy provider.
- [ ] observe when "autosubmit" is chosen the user is directed directly to VY instead of getting the "Log in to VY" button.

## Quality checks:

Please check these boxes to confirm this PR covers the following cases:

- Maintaining our upgrade path is essential. Check one or the other:
  - [ ] This PR  provides updates via `hook_update_N` or other means.
  - [ ] No updates are necessary for this change.
- Front end fixes should be tested against all of the Open Y Themes.
  - [ ] Tested against Carnation
  - [ ] Tested against Lily
  - [ ] Tested against Rose
  - [ ] This change does not contain front-end fixes.
- [ ] I have flagged this PR "Needs Review" or pinged the VY devs/QA
  team in Slack

### CODE REVIEW

File: modules/openy_gc_auth/modules/openy_gc_auth_example/openy_gc_auth_example.libraries.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+openy_gc_auth_example:
+  js:
```
Comment (podarok) [inline modules/openy_gc_auth/modules/openy_gc_auth_example/openy_gc_auth_example.libraries.yml:2]: We should always have version for all libraries - see https://www.drupal.org/node/3281083

File: modules/openy_gc_auth/modules/openy_gc_auth_example/openy_gc_auth_example.libraries.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    js/openy_gc_auth_example.js: {}
```
Comment (podarok) [inline modules/openy_gc_auth/modules/openy_gc_auth_example/openy_gc_auth_example.libraries.yml:3]: let's ensure we have proper options - minified and load - see https://www.drupal.org/node/3281085

**Review by podarok — CHANGES_REQUESTED** [review]

File: modules/openy_gc_auth/modules/openy_gc_auth_example/openy_gc_auth_example.libraries.yml (hunk 1 above)
Comment (froboy) [reply modules/openy_gc_auth/modules/openy_gc_auth_example/openy_gc_auth_example.libraries.yml:3]: Thanks @podarok.

File: modules/openy_gc_auth/modules/openy_gc_auth_example/openy_gc_auth_example.libraries.yml (hunk 2 above)
Comment (froboy) [reply modules/openy_gc_auth/modules/openy_gc_auth_example/openy_gc_auth_example.libraries.yml]: Added. Thanks.

**Pull request merged**